.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+#### What
+Please provide a short description of the purpose of this pull request.
+
+#### Why
+Explain why the proposed change is necessary.
+
+`Task`: https://omnivector-solutions.monday.com/boards/XXXX/pulses/YYYY
+
+---
+
+#### WIP PR
```

Comment:
Do you feel like this should be in the documentation on development workflow rather then the PR template?